### DISCUSSION TITLE
Part of DAW is shown and not the actual plugin.

### DISCUSSION THREAD

**Discussion by Glenson1** [apohl79/audiogridder#394] [Q&A] [answered]
Hello all,

When I load the latest version of Audiogridder on my mac os High Sierra (latest version) in Ableton and I try to open a plugin Audiogridder shows a part of Ableton in its window and not the plugin I try to load. 

Anyone else had this problem? And how can I fix this? What am I doing wrong?

Thanks for your help?
Glenn

**Comment by apohl79** [comment, marked as answer]
Sounds like you are running server and DAW on the same machine. You should probably disable screen capturing.

**Reply by Glenson1** [reply]
Yes, you are correct!

I will try: disable screen capturing.

Thank you :-)

**Reply by Glenson1** [reply]
Yes, that works. Thanks again! :-)
